activiti-cloud-query-service/activiti-cloud-starter-query/src/test/resources/application.properties

```diff
@@ -36,5 +36,4 @@ keycloak.security-constraints[0].securityCollections[0].patterns[0]=/v1/*
 keycloak.security-constraints[1].authRoles[0]=ACTIVITI_ADMIN
 keycloak.security-constraints[1].securityCollections[0].patterns[0]=/admin/*
 
-
-
+activiti.cloud.query.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service